sklearn/metrics/_plot/precision_recall_curve.py

```diff
@@ -118,6 +135,24 @@ def plot(self, ax=None, *, name=None, **kwargs):
             Name of precision recall curve for labeling. If `None`, use
             `estimator_name` if not `None`, otherwise no labeling is shown.
 
+        plot_chance_level : bool, default=False
+            Whether to plot the chance level.
+
+            .. versionadded:: 1.3
+
+        pos_prevalence : float, default=None
+            The prevalence of the positive label. It is used for plotting the
+            chance level line. If `plot_chance_level=True`, it must be provided
+            as a float between 0 and 1.
+
+            .. versionadded:: 1.3
```

Comment:
This value should be an attribute passed during in `__init__`. I don't think we want the user to tune it.